### PR TITLE
add babel, add tests for functions

### PR DESCRIPTION
**Pull request by AlexeyDolya**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,15 +4,24 @@
   "description": "learning js",
   "main": "index.js",
   "dependencies": {
+    "babel-jest": "^23.0.0-alpha.0",
```
Comment (LimarenkoDenis) [inline package.json]: use strict versions (without ^)

File: src/test/min.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export function min(a, b) {
+  if (a < b) {
+    return a;
+  } else {
```
Comment (LimarenkoDenis) [inline src/test/min.js]: the same comment 
```
if (a < b) {
+    return a;
+  }
return b
delete else
``

File: src/test/checkAge.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+export function checkAge(age) {
+  if (age > 18) {
+    return true;
+  }
+  return console.log('Родители разрешили?');
```
Comment (LimarenkoDenis) [inline src/test/checkAge.js]: maybe let's return false

File: src/test/checkAge.test.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import { checkAge } from './checkAge';
+
+test('if age is more', () => {
+  expect(checkAge(20)).toBeTruthy();
+});
+
+test('if age is less', () => {
+  expect(checkAge(10)).toBeFalsy();
```
Comment (LimarenkoDenis) [inline src/test/checkAge.test.js]: toBeFalsy return undefined?  compare with strict false is better probably

File: src/test/pow.test.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import { pow } from './pow';
+
+test('2 in the 5 degree', () => {
+  expect(pow(2, 5)).toBe(64);
+});
+
+test('10 in the -1 degree', () => {
+  expect(pow(10, -1)).toBe(10);
```
Comment (LimarenkoDenis) [inline src/test/pow.test.js]: WTF? 10 ^ -1 check please

File: src/test/checkAge.test.js (hunk 4 above)
Comment (AlexeyDolya) [reply src/test/checkAge.test.js]: in JS, there are six falsy values: false, 0, ''", null, undefined, and NaN. And toBeFalsy make adduction for all thes to boolean context

File: package.json [hunk 6]
```diff
@@ -4,15 +4,24 @@
...
     "eslint": "^4.19.0",
     "eslint-plugin-import": "^2.9.0",
-    "eslint-plugin-json": "^1.2.0"
+    "eslint-plugin-json": "^1.2.0",
```
Comment (LimarenkoDenis) [inline package.json]: fix versions

File: src/test/Loops/evenNum.js [hunk 7]
```diff
@@ -0,0 +1,11 @@
+export function evenNum(i, a) {
+  let arr = [];
+  for (i = 0; i < a; i++) {
+    if (i % 2 !== 0) continue;
```
Comment (LimarenkoDenis) [inline src/test/Loops/evenNum.js]: add `{}` to if construction

File: src/test/Loops/evenNum.js [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+    {
```
Comment (LimarenkoDenis) [inline src/test/Loops/evenNum.js]: align code

**Pull request merged**